website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -326,6 +326,12 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              _component_tags
 		}
+		buffer_send_duration_max_seconds: {
```

Comment:
I'm debating whether this should be buffer specific or something more "user-friendly" and opinionated like `component_backpressure_seconds`.  The later obfuscates what it is measuring but is more self-documenting in terms of how to interpret the value.

Reply:
I think calling it backpressure is just slightly off, because there's always some inherent latency between an item being sent from a component and being pushed into the buffer.

If anything, I'd say `component_event_send_duration_seconds`.

Reply:
Yeah, I agree. I'll leave it as is.